extensions/apps/antenna/src/components/reflect-editor/index.tsx

```diff
@@ -85,32 +88,42 @@ const ReflectEditor: React.FC<ReflectEditorProps> = props => {
   }, []);
 
   /*
-   * Currently jsdom doesn't support contenteditable and as a result slate editor as well.
+   * Currently jsdom doesn't support contenteditable and as a result slate editor can't be tested in jsdom.
    * This effect is a workaround to set the value of the editor during tests
    **/
   React.useEffect(() => {
     const editorValueForTest = getEditorValueForTest();
     if (editorValueForTest) {
-      setEditorState([
-        {
-          type: 'paragraph',
-          children: [
-            {
-              text: editorValueForTest,
-            },
-          ],
-        },
-      ]);
+      setEditorState(
```

Comment:
I'm not convinced that this is the way to do it. Maybe it was better for this component to allow an intialState of slate content type. Please create an issue with this to be addressed later.